src/pages/Navbar/Logo.jsx

```diff
@@ -0,0 +1,14 @@
+import { FireFilled } from '@ant-design/icons';
+
+const Logo = () => {
+  return (
+    <div className="logo">
+      <div className="logo-icon">
+        <FireFilled />
```

Comment:
can we use the same logo?

Reply:
do you mean to change the name of the class?

Reply:
I mean this one
![image](https://github.com/hasadna/open-bus-map-search/assets/11145132/ab7d86c9-51b4-407f-846d-37dc70bcfba3)

Reply:
yes, I changed it, I need the logo image.